cmd/kubeadm/app/preflight/checks.go

```diff
@@ -718,6 +746,15 @@ func RunInitMasterChecks(cfg *kubeadmapi.MasterConfiguration) error {
 		HTTPProxyCIDRCheck{Proto: "https", CIDR: cfg.Networking.PodSubnet},
 	}
 
+	if useCRI {
+		checks = append(checks, CRICheck{socket: criSocket, exec: execer})
+	} else {
+		// assume docker
+		checks = append(checks,
+			ServiceCheck{Service: "docker", CheckIfActive: true},
```

Comment:
any specific reason you didn't put all of this on a single line like the CRICheck above?

Reply:
nah, I can change it if you want

Reply:
please do

Reply:
fixed